File: 2031.py
for i in range(0, int(input())):
    jogador1 = input()
    jogador2 = input()
    if jogador1 == "ataque" or jogador2 == "ataque":
        if jogador1 == jogador2:
            print("Aniquilacao mutua")
        else:
            print("Jogador 1 venceu") if jogador1 == "ataque" else print("Jogador 2 venceu")
    elif jogador1 == "pedra" or jogador2 == "pedra":
        if jogador1 == jogador2:
            print("Sem ganhador")
        else:
            print("Jogador 1 venceu") if jogador1 == "pedra" else print("Jogador 2 venceu")
    else:
        print("Ambos venceram")

File: 2031.cpp
#include <iostream>
using namespace std;

int main(){
    int numero;
    string jogador1, jogador2;
    cin >> numero;
    for(int i = 0; i < numero; i++){
        cin >> jogador1;
        cin >> jogador2;
        if(jogador1 == "ataque" || jogador2 == "ataque"){
            if(jogador1 == jogador2)
                cout << "Aniquilacao mutua\n";
            else{
                if(jogador1 == "ataque")
                    cout << "Jogador 1 venceu\n"; 
                else
                    cout << "Jogador 2 venceu\n";
            }
        }else if(jogador1 == "pedra" || jogador2 == "pedra"){
            if(jogador1 == jogador2)
                cout << "Sem ganhador\n";
            else{
                if(jogador1 == "pedra")
                    cout << "Jogador 1 venceu\n";
                else
                    cout << "Jogador 2 venceu\n";
            }
        }else
            cout << "Ambos venceram\n";
    }
    return 0;
}
